Probesys/bileto
# Various improvements

## Related issue(s)



N/A

## Changes



See commits

## How to test manually



- create or update an orga role → check that "see all tickets" is renamed to "see all tickets of the organization"
- go to a ticket → check that "Actors" and "Priority" have better contrast
- open a ticket → check that the "emojis" button is not present anymore
- with a user with "update status" permission disabled, go to a ticket → check that "mark as solution" is no longer visible

## Checklist
